projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.spec.ts

```diff
@@ -554,6 +914,222 @@ describe('IgxSimpleCombo', () => {
             fixture.detectChanges();
             expect(document.activeElement).toEqual(addItemButton.nativeElement);
         }));
+
+        it('should close when an item is clicked on', fakeAsync(() => {
+            spyOn(combo, 'close').and.callThrough();
+            combo.open();
+            fixture.detectChanges();
+            const item1 = fixture.debugElement.query(By.css(`.${CSS_CLASS_DROPDOWNLISTITEM}`));
+            expect(item1).toBeDefined();
+
+            UIInteractions.simulateClickEvent(item1.nativeElement);
```

Comment:
simulateClickEvent uses dispatchEvent, which we try to avoid when possible. Use triggerEventHandler() instaed:
item1.triggerEventHandler('click', UIInteractions.getMouseEvent('click'));